tucson_teststand/loonie/launch_AuxTel_Illumination_containers.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Launch electrometer, atmonochromator, fiberspectrograph, on toonie
+
+# Get current working directory
+cwd=$(pwd)
+
+# Check to see if a name was given, otherwise assume both
+if [ $# -lt 1 ]; then
+  cscs="electrometer1 atmonochromator fiberspectrograph_broadband"
+else
+  cscs="$1"
+fi
+
+if [ ! -d ${OSPL_SHMEM_SOCKET_DIR} ] 
```

Comment:
This is only defined in the `setup.env` file, so if you execute this script without the envvar set it will fail which will happen from a fresh environment. The scripts in this repo generally assume the supporting infrastructure is available. There is a `setup_repos` script in the `bin` dir which can be leveraged to deal with machines that need a new installation if they haven't had one before.

Reply:
OK, so the setup.env file gets sourced a few lines down. Options are:
1. I can move it up here so it'll always be declared.
2. Move the creation of the directory into your setup-repo script... 

I'm tempted not to do the latter until we can test it.... which can't happen today :(